### DISCUSSION TITLE
How to design custom Homepage in Markdown?

### DISCUSSION THREAD

**Discussion by FuadEfendi** [razonyang/hugo-theme-bootstrap#699] [Q&A] [answered]
I found in documentation only this, https://hbs.razonyang.com/v1/en/docs/advanced/layouts-overriding/

`$ echo "MY CUSTOM HOMEPAGE" > layouts/index.html`

But how to create Homepage in markdown?

**Comment by razonyang** [comment]
See https://github.com/razonyang/hugo-theme-bootstrap/discussions/699#discussioncomment-3650083
~You can create a `content/index.md` file, so that the homepage will be rendered in the `single` page layout. I'm not sure if it meets your requirements.~

**Reply by FuadEfendi** [reply]
I asked because (unusually) neither `content/index.md` nor `content/_index.md` works with HBS...

**Reply by razonyang** [reply]
Would you mind explaining it in details? I didn't get it.
![image](https://user-images.githubusercontent.com/17720932/189495305-0a463ffc-33f3-4daa-8c31-d6126a90ac41.png)

**Comment by FuadEfendi** [comment]
I have `_index.md` in the `content` folder; renamed it to `index.md` and it works now (I forgot previously to reset `layouts`)

Thank you for confirming, you saved me many hours!!!

**Reply by razonyang** [reply]
You're welcome.

**Comment by razonyang** [comment, marked as answer]
It's able to custom home page by using [landing layout](https://hbs.razonyang.com/v1/en/docs/layouts/landing/) now. Please [upgrade](https://hbs.razonyang.com/v1/en/docs/getting-started/upgradation/) to the latest commit(`master` branch) if you want to use this feature.
And there be a problem if create `content/index.md`, the others posts will not be rendered, since the home page is treated as a single page, not a section.
